### ISSUE TITLE
ResourceWarning: unclosed file 

### ISSUE THREAD

**Issue by MB23** [elmirjagudin/symstore#10]
File is not being closed after reading the last transaction and there are open warnings in the format:

```
site-packages/symstore/symstore.py:403: ResourceWarning: unclosed file <_io.TextIOWrapper name='/tmp/000Admin/lastid.txt' mode='r' encoding='UTF-8'>
  last_id = int(open(last_id_file, "r").read())
ResourceWarning: Enable tracemalloc to get the object allocation traceback
```
This could be solved by closing file after read()

**Comment by elmirjagudin** [comment]
In what situation are you getting this warning? How are you running the code? Which python/os etc? 

Obviously the file is never closed, but I have never seen this warning. I want to figure out how to catch these warning with the tests.

**Comment by elmirjagudin** [comment]
Fixed in https://github.com/elmirjagudin/symstore/commit/e53c42216c74b8f246305bda62f0a1988b36e67c

The fix will be included into relase 0.2.6.
